packages/block-library/src/term-description/index.php

```diff
@@ -13,8 +13,9 @@
  * @return string Returns the filtered post content of the current post.
```

Comment:
Any chance you could correct the `@return` comment. eg

Returns the description of the current taxonomy term, if available